Sample/HttpCache.cs

```diff
@@ -17,32 +17,53 @@ public HttpCache()
 			_cache = HttpRuntime.Cache;
 		}
 
-		public object Add(string key, object entry, DateTime utcExpiry)
+		public object Add(string key, object entry, DateTime utcExpiry, string region = null)
 		{
 			return _cache.Add(key, entry, null, utcExpiry, Cache.NoSlidingExpiration, CacheItemPriority.Default, null);
 		}
 
-		public object Get(string key)
+		public object Get(string key, string region = null)
 		{
 			return _cache.Get(key);
 		}
 
-		public void Remove(string key)
+		public void Remove(string key, string region = null)
 		{
 			_cache.Remove(key);
 		}
 
-		public void Set(string key, object entry, DateTime utcExpiry)
+		public void Set(string key, object entry, DateTime utcExpiry, string region = null)
 		{
 			_cache.Insert(key, entry, null, utcExpiry, Cache.NoSlidingExpiration, CacheItemPriority.Default, null);
 		}
 
-		public void Clear()
+		public void Clear(string region = null)
 		{
 			var items = _cache.GetEnumerator();
 
 			while (items.MoveNext())
 				_cache.Remove(items.Key.ToString());
 		}
+
+		const string ChannelPrefix = "channel_";
+		public void Subscribe(string channel, Action<string, string> callback)

```

Comment:
This allows us to remove all timestamps-related staff. So we need it in trivial implementation.